package.json

```diff
@@ -18,6 +18,7 @@
     "@brainhubeu/react-carousel": "1.19.26",
     "@fortawesome/fontawesome-free": "^5.13.0",
     "@fortawesome/fontawesome-svg-core": "^1.2.4",
+    "@fortawesome/free-brands-svg-icons": "^5.15.2",
```

Comment:
This dep was the easiest way for me to get a Docker icon (that respects CSS `color`) to represent Docker Repo entities.